RELEASE-NOTES.txt

```diff
@@ -1,5 +1,21 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 
+17.2
+-----
+
+
+17.2
+-----
+
+
+17.2
+-----
+
+
```

Comment:
@JavonDavis Looks like there was an issue with the script and we have multiple `17.2` versions.

```suggestion
```

Reply:
Ahh good catch!